Base64/test/Base64Wrapper.test.ts

```diff
@@ -0,0 +1,114 @@
+import { ethers } from 'hardhat'
+import { expect } from 'chai'
+import web3 from 'web3'
+
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { ContractFactory, Contract } from 'ethers'
+import '@nomiclabs/hardhat-ethers'
+
+describe('Test Base64Wrapper', function () {
+    let owner: SignerWithAddress
+    let base64WrapperFactory: ContractFactory
+    let base64Wrapper: Contract
+
+    before(async function () {
+        // Getting the users provided by ethers
+        [owner] = await ethers.getSigners()
+
+        // Getting the Base64Wrapper contract code (abi, bytecode, name)
+        base64WrapperFactory = await ethers.getContractFactory('Base64Wrapper')
+
+        // Deploying the instance
+        base64Wrapper = await base64WrapperFactory.deploy()
+        await base64Wrapper.deployed()
+    })
+
+    it('check deployment', async function () {
+    })
```

Comment:
remove this part or add the function body

Reply:
Removed. Thank you.